models/junos/security_policies/junos_security_policies.yml

```diff
@@ -0,0 +1,541 @@
+---
```

Comment:
to use resource module builder `https://github.com/ansible-network/resource_module_builder` we still need to have header. You can take reference from https://github.com/ansible-network/resource_module_models/blob/master/models/junos/ntp_global/junos_ntp_global.yaml
---
GENERATOR_VERSION: '1.0'
ANSIBLE_METADATA: |
  {
      'metadata_version': '1.1',
      'status': ['preview'],
      'supported_by': 'network'
  }
NETWORK_OS: junos
RESOURCE: security_polices
COPYRIGHT: Copyright 2022 Red Hat